package.json

```diff
@@ -25,19 +25,19 @@
   "homepage": "https://github.com/conventional-changelog/conventional-changelog-angular#readme",
   "devDependencies": {
     "better-than-before": "^1.0.0",
-    "chai": "^3.4.1",
```

Comment:
These bumps are not necessary. I'm using `^` here so it will get the latest changes (plus they are just devDeps).